test/integration/backward_compatible/parallel/map/NearCachedMapTest.js

```diff
@@ -103,6 +103,7 @@ describe('NearCachedMapTest', function () {
                 }
 
                 await map1.get('key1');
+                expect(getNearCacheStats(map1).missCount).to.be.eq(1);
```

Comment:
This is a required precondition for the assertion in `assertTrueEventually` to be valid. I added it here to be sure this assertion holds.